File: Java/1760/A.java
import java.util.*;

public class A {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int t = in.nextInt();
		for(int i = 0;i<t;i++) {
			int[] nums = new int[3];
			nums[0] = in.nextInt();
			nums[1] = in.nextInt();
			nums[2] = in.nextInt();
			Arrays.sort(nums);
			System.out.println(nums[1]);
		}
	}
}

File: C++/1760/A.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(){
    int i, j, n;
    int nums[3];
    cin >> n;
    for(i = 0;i<n;i++){
        for(j = 0;j<3;j++){
            cin >> nums[j];
        }
        sort(nums,nums+3);
        printf("%d\n",nums[1]);
    }
    return 0;
}
